src/SprykerEco/Zed/Heidelpay/Persistence/HeidelpayEntityManager.php

```diff
@@ -27,7 +27,6 @@ public function savePaymentHeidelpayDirectDebitRegistrationEntity(
     ): HeidelpayDirectDebitRegistrationTransfer {
         $paymentHeidelpayDirectDebitRegistrationEntity = $this->getPaymentHeidelpayDirectDebitRegistrationQuery()
             ->filterByRegistrationUniqueId($directDebitRegistrationTransfer->getRegistrationUniqueId())
-            ->filterByFkCustomerAddress($directDebitRegistrationTransfer->getIdCustomerAddress())
```

Comment:
same question

Reply:
A bug was discovered during tests implementation. In previous version I had duplication of registration rows after updating row with customer address.